safesens/device/schema.py

```diff
@@ -18,5 +18,11 @@ class Meta:
         interfaces = (graphene.relay.Node, )
 
 class DeviceQuery(graphene.ObjectType):
-    device = graphene.relay.Node.Field(DeviceNode)
-    all_devices = DjangoFilterConnectionField(DeviceNode, filterset_class=DeviceFilter)
+    user_devices = DjangoFilterConnectionField(DeviceNode, filterset_class=DeviceFilter)
+
+    def resolve_user_devices(self, info, **kwargs):
+        current_user = info.context.user
+        if current_user.is_authenticated:
```

Comment:
This also looks good. Just add the GraphQLError for when the user is not authenticated. and remove the comment at the end.